sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
@JackKrupansky, there is already a way to show the precise isolation being actually used. `SHOW TRANSACTION ISOLATION LEVEL` does it:

```
SET TRANSACTION ISOLATION LEVEL READ COMMITTED

SHOW TRANSACTION ISOLATION LEVEL
----
SNAPSHOT
```